spec/features/listing_property_spec.rb

```diff
@@ -0,0 +1,16 @@
+#require 'spec_helper'
+
+feature 'As a user' do
+  scenario 'I can list a property' do
+    visit ('/property')
+    click_button('List Property')
+    expect(current_path).to eq '/property/new'
+    fill_in :name, with: "Imm's Inn"
+    # fill_in :description, with: "Very Nice"
```

Comment:
Let's bring the commented out code back in until we have spoken with Mary again!